google/cloud/monitoring_v3/query.py

```diff
@@ -334,20 +334,20 @@ def align(self, per_series_aligner, seconds=0, minutes=0, hours=0):
 
         Example::
 
-            from google.cloud.monitoring import enums
+            import google.cloud.monitoring as monitoring
```

Comment:
```suggestion
            from google.cloud import monitoring
```
nit: This style is more common in samples and docs